.editorconfig

```diff
@@ -119,6 +119,13 @@ csharp_prefer_simple_default_expression = true:suggestion
 
 dotnet_code_quality_unused_parameters = non_public:suggestion
 
+# Dotnet diagnostic settings:
+[*.cs]
+
+# CA1859: Use concrete types when possible for improved performance
+# https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/quality-rules/ca1859
+dotnet_diagnostic.CA1859.severity = suggestion
+
```

Comment:
@iSazonov We could probably move all our existing diagnostic rules from globalconfig here in another PR.

Reply:
Yes, we can. 
Looking commit history of the file we can figure out all the changed options.

Reply:
And please add dummy change in a file to trigger CIs.

Reply:
Added another dot to build.psm1.